src/ircd_parser.y

```diff
@@ -543,6 +545,73 @@ serverinfo_ssl_certificate_file: SSL_CERTIFICATE_FILE '=' QSTRING ';'
       yyerror("RSA private key does not match the SSL certificate public key!");
       break;
     }
+
+    if (ServerInfo.dh_params != NULL &&
+      !SSL_CTX_set_tmp_dh(ServerInfo.ctx, ServerInfo.dh_params))
+    {
+      yyerror(ERR_lib_error_string(ERR_get_error()));
+    }
+  }
+#endif
+};
+
+serverinfo_dh_params_file: DH_PARAMS_FILE '=' QSTRING ';'
+{
+#ifdef HAVE_LIBCRYPTO
+  if (ypass == 1)
+  {
+    BIO *file;
+    int codes;
+
+    if (ServerInfo.dh_params)
```

Comment:
This is nit picky I know, but I do rather prefer if statements to have 'proper' boolean expressions, ie if(foo != NULL) rather than if(foo)